.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--
+Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+
+Please make sure that you are familiar with and follow the Code of Conduct for
+this project (found in the CODE_OF_CONDUCT.md file).
+
+Also, please make sure you're familiar with and follow the instructions in the
+contributing guidelines (found in the CONTRIBUTING.md file).
+
+If you're new to contributing to open source projects, you might find this free
+video course helpful: https://kcd.im/pull-request
+
+Please fill out the information below to expedite the review and (hopefully)
+merge of your pull request!
+-->
+
+## Reasoning 💡
+
+<!--
+What changes are being made? What feature/bug is being fixed here?
+
+IMPORTANT: Which, if any, adapter is being affected? Or are you wanting to add
+a new one?
```

Comment:
```suggestion
```
Sorry, I missed that in my previous review. This whole template could probably be simplified a lot. Doc changes are not that important to follow a template in my opinion.

Reply:
Yup, simplified this as well :+1: